externs/browser/intersection_observer.js

```diff
@@ -127,7 +127,7 @@ function IntersectionObserver(handler, opt_options) {};
  * The root Element to use for intersection, or null if the observer uses the
  * implicit root.
  * @see https://w3c.github.io/IntersectionObserver/#dom-intersectionobserver-root
- * @const {?Element}
+ * @const {?Element|null}
```

Comment:
?Element means that it includes null, so ?Element|null is redundant.

Reply:
Thanks for noting! Reverted it back.